mlir/test/Dialect/Linalg/roundtrip.mlir

```diff
@@ -4,7 +4,7 @@
 // TODO: Re-enable LLVM lowering test.
 //
 // Test that we can lower all the way to LLVM without crashing, don't check results here.
-// DISABLED: mlir-opt %s -='use-opaque-pointers=1' -o=/dev/null 2>&1
+// DISABLED: mlir-opt %s -o=/dev/null 2>&1
```

Comment:
This line looked already broken in main. Maybe we should just remove this run line + the TODO above?

Reply:
@nicolasvasilache should we just remove this line in a followup patch?